validator/testcases/javascript/predefinedentities.py

```diff
@@ -64,6 +101,12 @@
 ]
 
 
+def is_shared_scope(t, r=None, rn=None):
```

Comment:
Is that clearer for you @mstriemer ? I'm still feeling confused about all this, but I don't have any context, and don't know the amo-validator code.